Tests/Expected/XCAssets/swift4-context-all-customname.swift

```diff
@@ -0,0 +1,133 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+#if os(OSX)
+  import AppKit.NSImage
+  typealias Color = NSColor
+  typealias Image = NSImage
+#elseif os(iOS) || os(tvOS) || os(watchOS)
+  import UIKit.UIImage
+  typealias Color = UIColor
+  typealias Image = UIImage
+#endif
+
+// swiftlint:disable file_length
+
+@available(*, deprecated, renamed: "XCTImageAsset")
+typealias XCTAssetsType = XCTImageAsset
+
+struct XCTImageAsset {
+  fileprivate var value: String
+
+  var image: Image {
+    let bundle = Bundle(for: BundleToken.self)
+    #if os(iOS) || os(tvOS)
+    let image = Image(named: value, in: bundle, compatibleWith: nil)
+    #elseif os(OSX)
+    let image = bundle.image(forResource: NSImage.Name(value))
+    #elseif os(watchOS)
+    let image = Image(named: value)
+    #endif
+    guard let result = image else { fatalError("Unable to load image \(value).") }
+    return result
+  }
+}
+
+struct XCTColorAsset {
+  fileprivate var value: String
```

Comment:
Ah. Meaning one color entry in the Asset Catalog can have multiple values, because Normal vs. P3 or because they can have different actual colors for different trait collections etc, is that it?
Damn.